docs/docs/software-architecture.md

```diff
@@ -31,139 +31,147 @@ UI - User Interface
-| Atomic cross-chain swap | Two users want to exchange BTC and BTS | The user submits an exchange order | User 1 and User 2 | None             | - Users should be in possession of the cryptocurrency that is desired by the other party - Users should open a HTLC in their respective Blockchains | - Validating the user - Hash and time lock conditions should be met | - Submit desired order - Start a HTLC - Fund and redeem a swap - Key pair generator - Verification process of validity between private and public keys - Transaction successful or funds are refunded | - Network failure - App crashing - Too much market volatility |
+| Atomic cross-chain swap | Two users want to exchange BTC and BTS | Two HTLCs are created in the respective blockchains which locked by the same secret hash provided by the proposer/proposing party | User 1 and User 2 | None             | - Users should be in possession of the cryptocurrency that is desired by the other party     - Users should create an HTLC in their respective Blockchains by signing their transactions | - Validating the user     - Hash and time lock conditions should be met | - Start an HTLC     - Fund and redeem a swap     - Key pair generator     - Verification process of validity between private and public keys     - Transaction successful or funds are refunded | - Network failure   - App crashing   - Too much market volatility |
 
+   
 ## 5. Architectural Goals and Constraints
 
-There are some key requirements and system constraints that have a significant bearing on the architecture. They are:ƒ
+There are some key requirements and system constraints that have a significant bearing on the architecture. They are:
 
 - The Swapchain platform should satisfy all requirements set by the AMOS project, i.e. licenses, schedule, tools, etc.
 - The BOM should be within a reasonable amount.
 - Swapchain platform should be accessible with local and remote PCs.
 - All private and public information of the clients using the platform should be secured safely.
-- The Swapchain platform will be implemented as a client-server system. The client portion resides on PCs and the server portion must operate on the APIs and databases used.
+- The Swapchain platform will be implemented as a client-server system. The client portion resides on PCs and the server portion must operate on the APIs used.
 
+   
 ## 6. Deployment
 
-This UI is a system which is hosted in an API. The database will be hosted by ChainSquad once the software is delivered. Atomic transactions are conducted in the backend, so that the client computer does not have to spend much of CPU power.
-
+This UI is an API which is hosted in Netlify. After finishing the AMOS course and therewith the project, the software will be hosted by ChainSquad. Atomic transactions are conducted in the backend, so that the client computer does not have to spend much of CPU power.
```

Comment:
The UI is hosted on netlify. For users interested in the CLI, they can download the repository from github and build it from source, documentation is provided in the README.